torch/_inductor/lowering.py

```diff
@@ -347,14 +347,12 @@ def const_func(x):
     ex = next(x for x in inputs if isinstance(x, (TensorBox, ExpandView)))
     out = []
     for x in inputs:
-        if isinstance(x, (int, float)):
+        if isinstance(x, (int, float, sympy.Expr)):
             out.append(
                 ExpandView.create(
                     ir.Constant(x, ex.get_dtype(), ex.get_device()), list(ex.get_size())
                 )
             )
-        elif isinstance(x, sympy.Expr):
-            out.append(IndexingConstant(x, ex.get_dtype(), ex.get_device()))
```

Comment:
You still need to create an `IndexingConstant` node here, just wrap it in `ExpandView.create`.

Reply:
Oh, I see. Thanks for the help, @peterbell10.